### PR TITLE
Plot resurrection. Night & semester. Dark & light mode. Progressive plotting for semesters.

### PR DESCRIPTION
**Pull request by rpiaggio**
Kapture 2021-08-04 at 18 21 24

Kapture 2021-08-04 at 18 22 30

(Commented code regarding `constraintSetIds` is unrelated and left on purpose.)

### CODE REVIEW

**Comment by cquiroz** [comment]
Cool screenshot

File: explore/src/main/scala/explore/targeteditor/SkyPlotSection.scala [hunk 1]
```diff
@@ -63,19 +81,38 @@ object SkyPlotSection {
...
+      implicit val ctx = props.ctx
+
       <.div(ExploreStyles.SkyPlotSection)(
-        <.div(ExploreStyles.SkyPlot)(
-          SkyPlot(state.site, props.coords, state.date, state.zoneId, 350)
-        ),
+        <.div(ExploreStyles.SkyPlot) {
+          state.plotPeriod match {
+            case PlotPeriod.Night    =>
+              SkyPlotNight(state.site, props.coords, state.date, state.zoneId, 350)
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/SkyPlotSection.scala:103]: What's the 350?

**Review by cquiroz — APPROVED** [review]
Looks great, thanks. Would you verify it works in production, sometimes we have troubles with third party libs

**Comment by rpiaggio** [comment]
> Looks great, thanks. Would you verify it works in production, sometimes we have troubles with third party libs

Sure, I'll deploy a review app

File: explore/src/main/scala/explore/targeteditor/SkyPlotSection.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/SkyPlotSection.scala:103]: That's the chart height, in pixels.

**Comment by rpiaggio** [comment]
> Cool screenshot

Thank you. Actually thanks to the screenshot I just noticed that the twilight band is not visible in dark mode, I'll fix that.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**